src/assets/browser/app/src/composit/stage/position.rs

```diff
@@ -1,6 +1,6 @@
 use std::fmt;
 
-use composit::Zoom;
+use super::zoom::Zoom;
```

Comment:
(Whole file) separate "stage" position management into own module. Only lightly coupled with rest of package and easier to factor when methods are private to a small module.